packages/compiler/src/compilation/setup-compilation.ts

```diff
@@ -3,7 +3,14 @@ import * as compilerCli from '@angular/compiler-cli';
 import * as ts from 'typescript';
 import { compileStringAsync } from 'sass-embedded';
 import { augmentHostWithResources } from './augments';
-import { PluginAngularOptions } from '../../models/plugin-options';
+import { InlineStyleExtension, FileReplacement } from '../models';
+
+export interface SetupCompilationOptions {
+  tsconfigPath: string;
+  jit: boolean;
+  inlineStylesExtension: InlineStyleExtension;
+  fileReplacements: Array<FileReplacement>;
+}
```

Comment:
you could derive the type from `AngularCompilerOptions` and Typescript `CompilerOptions`